src/main/java/com/springweb/dv_spring_web_mongo/dto/ProjectDTO.java

```diff
@@ -1,49 +1,22 @@
 package com.springweb.dv_spring_web_mongo.dto;
 
 import com.springweb.dv_spring_web_mongo.model.Project;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 import org.springframework.data.annotation.Id;
-import org.springframework.data.mongodb.core.mapping.Document;
 
-public class ProjectDTO {
-
-    public ProjectDTO() {
-    }
-
-    public ProjectDTO(String id, String projectName) {
-        this.id = id;
-        this.projectName = projectName;
-    }
+import javax.validation.constraints.NotNull;
 
-    @Id
```

Comment:
It is not good idea to remove `id` field from here because the dto object is used in GET /api/projects method and it is important to have this field in result of method